libs/MeshKernelApi/include/MeshKernelApi/Utils.hpp

```diff
@@ -421,10 +425,11 @@ namespace meshkernelapi
         {
             yCoordinates[i] = griddedSamples.y_coordinates[i];
         }
-        return std::make_unique<meshkernel::BilinearInterpolationOnGriddedSamples<ValueType>>(mesh2d,
-                                                                                              xCoordinates,
-                                                                                              yCoordinates,
-                                                                                              griddedSamples.values);
+        return std::make_unique<meshkernel::BilinearInterpolationOnGriddedSamples<InterpolatableType>>(mesh2d,
```

Comment:
Just a minor point of interest, should we also use the std::span for the x- and y-coordinate arrays. This will also save a little storage.

Reply:
std::span has a size method. Can use that.

Reply:
can do

Reply:
Good point. Done.